blathering-barnacles/Bangazon-2
# Jb reg customer

Description:
Added functionality to create a customer at the same time a user is registered so that they have the same id. I did this by refactoring the registration view to use the new user id to pass in as the primary key and user foreign key when creating a new customer.

Steps to Test:
Go to localhost:8000
Logout
Register a new user
Look in the database and look at the customer table. Make sure the primary key is the same as the user id foreign key
